staging/src/k8s.io/apiserver/pkg/endpoints/apiserver_test.go

```diff
@@ -3865,55 +3865,6 @@ func (storage *SimpleRESTStorageWithDeleteCollection) DeleteCollection(ctx conte
 	return nil, nil
 }
 
-func TestDryRun(t *testing.T) {
```

Comment:
could keep this to make sure things still respect the feature gate being disabled until it is removed (with a call to `defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.DryRun, false)()`)

Reply:
Yes, agreed, makes sense to me